src/coreclr/jit/gentree.h

```diff
@@ -3279,6 +3283,149 @@ struct GenTreeStrCon : public GenTree
 #endif
 };
 
+/* gtVecCon -- vector  constant (GT_CNS_VEC) */
+
```

Comment:
```suggestion
// GenTreeVecCon -- vector  constant (GT_CNS_VEC)
//
```
Both `gtXYZ` and `/* */` comments are considered legacy.

Reply:
Fixed.